app/routes/browse.js

```diff
@@ -12,7 +12,7 @@ export default Ember.Route.extend({
     model() {
         return Ember.RSVP.hash({
             preprints: this.store.findAll('preprint'),
-            taxonomy: this.store.find('taxonomy', 1)
+            taxonomy: this.store.findAll('taxonomy')
```

Comment:
So the on the browse route we are returning `taxonomy: this.store.findAll('taxonomy')` and for the index route, `subjects: this.store.findAll('taxonomy')`. This is confusing.